dm/dm/master/scheduler/scheduler.go

```diff
@@ -674,17 +676,79 @@ func (s *Scheduler) TransferSource(source, worker string) error {
 		}
 		return true
 	})
-
 	if len(runningTasks) > 0 {
-		return terror.ErrSchedulerRequireNotRunning.Generate(runningTasks, source)
+		queryWorkerF := func() (*workerrpc.Response, error) {
```

Comment:
How about move this function closure to scheduler/worker.go as a worker rpc method, we can reuse it and improve readability of TransferSource

Reply:
split func to https://github.com/pingcap/tiflow/commit/f0bff3cb24606bb0f262a63a1f25c6a6662260c1 ptal 🧡